src/backend/expungeservice/form_filling.py

```diff
@@ -178,8 +181,9 @@ def _build_pdf_for_case(
         else:
             return None
 
-    @staticmethod
+    @classmethod
```

Comment:
Nitpick. I prefer to keep it staticmethod. Marking a function as a classmethod suggests to me that you're doing something more complicated with the cls but for just a constant, I think it adds more noise than helps. You can just have it top-level for the `CHECK_MARK` and just do `"has_conviction": CHECK_MARK if has_conviction else "",`

Reply:
Good to know. My Python is rusty so that was the first thing to come to mind when trying to pass that constant around.